en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -286,141 +201,98 @@ hosting accountrecoveryendpoint.war also on a separate web server.
-        and, public cert of `           <          `
-        `           IS_HOME>/repository/resources/security/wso2carbon.jks          `
-        into tomcat’s truststore.
+        need to import tomcat keystore’s public cert in to: `<IS_HOME>/repository/resources/security/client-truststore.jks` and, public cert of `<IS_HOME>/repository/resources/security/wso2carbon.jks` into tomcat’s truststore.
     
 
-4.  Open `           <TOMCAT_HOME>/bin/catalina.sh          ` and add
-    following JAVA\_OPTS.
+4.  Open `<TOMCAT_HOME>/bin/catalina.sh` and add following JAVA\_OPTS.
 
     ``` xml
     JAVA_OPTS="$JAVA_OPTS --Djavax.net.ssl.keyStore=$IS_HOME/repository/resources/security/wso2carbon.jks -Djavax.net.ssl.keyStorePassword=wso2carbon"
     JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.trustStore=$IS_HOME/repository/resources/security/client-truststore.jks -Djavax.net.ssl.trustStorePassword=wso2carbon"
     ```
 
-5.  Run `          setup-authentication.sh         ` obtained from [step
-    2](#HostingAuthenticationEndpointonaDifferentServer-step2) and
-    follow the instructions.
-6.  Once the script is complete, then the authentication endpoint is set
-    up in the given `          <TOMCAT_HOME>/webapps         ` location.
-7.  Uncomment following section in
-    `           <TOMCAT_HOME>/webapps/authenticationendpoint/WEB-INF/web.xml          `
-    file and point to identity server URLs.
+5.  Run `setup-authentication.sh` obtained from [step 2](#HostingAuthenticationEndpointonaDifferentServer-step2) and follow the instructions.
+
+6.  Once the script is complete, then the authentication endpoint is set up in the given `<TOMCAT_HOME>/webapps` location.
+
+7.  Uncomment following section in `<TOMCAT_HOME>/webapps/authenticationendpoint/WEB-INF/web.xml` file and point to identity server URLs.
 
     ``` xml
-        …...   
+    …...   
+    <context-param>
+            <param-name>IdentityManagementEndpointContextURL</param-name>
+    <param-value>https://localhost:9443/accountrecoveryendpoint</param-value>
+        </context-param>
         <context-param>
-               <param-name>IdentityManagementEndpointContextURL</param-name>
-        <param-value>https://localhost:9443/accountrecoveryendpoint</param-value>
-           </context-param>
-            <context-param>
-               <param-name>AccountRecoveryRESTEndpointURL</param-name>
-             <param-value>https://localhost:9443/t/tenant-domain/api/identity/user/v0.9/</param-value>
-           </context-param>
-        …..
-            <context-param>
-                <param-name>IdentityServerEndpointContextURL</param-name>
-                <param-value>https://localhost:9443</param-value>
-            </context-param>
-        …...
+            <param-name>AccountRecoveryRESTEndpointURL</param-name>
+            <param-value>https://localhost:9443/t/tenant-domain/api/identity/user/v0.9/</param-value>
+        </context-param>
```

Comment:
```<context-param>
        <param-name>AuthenticationRESTEndpointURL</param-name>
        <param-value>https://is-dev.cool.com:9444/api/identity/auth/v1.1/</param-value>
    </context-param>```

Reply:
since we've used `localhost:9443' instead of `is-dev.cool.com:9444` in all the other param values, would it be correct to change this to :
<param-value>https://localhost:9443/api/identity/auth/v1.1/</param-value>

Reply:
Yes. Sorry, my intention was to correct the API version in the URL